CHANGELOG.md

```diff
@@ -1,16 +1,3 @@
-## Solidus 3.3.0 (master, unreleased)
-- Introduced level enum to `Spree::TaxRate` to allow for order-level taxes [#4491](https://github.com/solidusio/solidus/pull/4491) ([adammathys](https://github.com/adammathys))
-- Added `Spree::Calculator::FlatFee` tax calculator [#4491](https://github.com/solidusio/solidus/pull/4491) ([adammathys](https://github.com/adammathys))
-
-### Other Important Changes
-
-#### Colorado Delivery Fee
-
-With the introduction of the `Spree::Calculator::FlatFee` and `Spree::TaxRate#level`, we now have the ability to support the new Colorado Delivery Fee. There's also a new rake task provided to help setup the requisite models to support the fee: 
-```bash
-rake taxes:colorado_delivery_fee
-```
-
```

Comment:
I don't think so. It's part of the same logical change. We don't understand the update of the Changelog without the context of it being automatically generated from that moment on. Does it make sense?

Reply:
@waiting-for-dev Hmmm... It seems you _manually_ removed these sections (Solidus 3.3.0, Other Important Changes, Colorado Delivery Fee) from the changelog because they would conflict with any future runs of the `prepare_release` action. Is that correct?